book/src/filters.md

```diff
@@ -62,6 +63,23 @@ Output:
 2
 ```
 
+### as_ref
+[#as_ref]: #as_ref
+
+Put the given argument behind a reference.
```

Comment:
Okay, one more nit: "Put .. behind a reference" doesn't seem like language that is commonly used for this? I'd describe it as maybe "Creates a reference to ..".

Reply:
:+1: